### PR TITLE
Onboarding wizard enhancements

### PR DESCRIPTION
**Pull request by Azanul**
## Problem

- Onboarding wizard doesn't schedule the fetching of resources
- No support for opencost in onboarding wizard

## Solution

- Used gocron to fetch resources for added account every hour
- New input field in kubernetes part of onboarding wizard

## Screenshots

Screenshot 2024-01-09 at 1 19 14 PM

## Checklist

- [X] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [X] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [X] Any dependencies have been added to the project, if necessary

## Reviewers

@jakepage91 @mlabouardy @AllieMendes

### CODE REVIEW

File: handlers/accounts_handler.go [hunk 1]
```diff
@@ -112,7 +114,17 @@ func (handler *ApiHandler) NewCloudAccountHandler(c *gin.Context) {
 
 		accountId, _ := result.LastInsertId()
 		account.Id = accountId
-		go fetchResourcesForAccount(c, account, handler.db, []string{})
+
+		cron := gocron.NewScheduler(time.UTC)
+		_, err = cron.Every(1).Hours().Do(func() {
```
Comment (AvineshTripathi) [inline handlers/accounts_handler.go:119]: Wouldn't this be another cron that keeps on running. Lets say a new acc was added and before that the main core cron was already running now with the addition of new clou acc this cron starts too every other hour for same acc. Wont this be redundant

Comment (Azanul) [reply handlers/accounts_handler.go:119]: main cron doesn't have the acc covered by this one

Comment (AvineshTripathi) [reply handlers/accounts_handler.go:119]: In case of multiple addition of cloud acc there will be multiple crons running

Comment (AvineshTripathi) [reply handlers/accounts_handler.go:119]: sure for time being we can use it but definately we can improve this a lot

Comment (Azanul) [reply handlers/accounts_handler.go:119]: Indeed

**Review by AvineshTripathi — APPROVED** [review]

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**